src/bashly.yml

```diff
@@ -42,6 +41,26 @@ commands:
         required: true
     examples:
       - dcu logo coreboot.rom -l bootsplash.bmp
+  - name: smmstore
+    help: Edit settings stored in SMMSTORE
+    args:
+      - name: dasharo_rom_file
+        required: true
+        help: Dasharo firmware file (e.g. coreboot.rom)
+    flags:
+      - long: --get
+        arg: var
+        help: Name of an SMMSTORE variable
+      - long: --list
+        help: List available variables
+      - long: --set
+        arg: var
+        help: Name of an SMMSTORE variable
```

Comment:
To be updated once we have a better nomenclature

Reply:
https://github.com/Dasharo/dcu/pull/5/commits/2cf531f645fc532b64383b6104c3be1b6fba0c75